apertium-tur.tur.lexc

```diff
@@ -25077,7 +25080,7 @@ devrol:devrol V-IR-IV ; ! "devolve on" (DIR)
 didin:didin V-IR-IV ; ! "moil" (DIR)
 din:din V-AR-IV ; ! "quiet down" (DIR)
 diril:diril V-IR-IV ; ! "revive" (DIR)
-doğ:doğ V-AR-IV ; ! "born" (DIR)
+doğ:doğ V-AR-TV ; ! "born" (DIR)
 doğrul:doğrul V-IR-IV ; ! "straighten up" (DIR)
```

Comment:
TV

Reply:
Yes, TV, so was that correct?, Or did you mean to write IV?.